contracts/proxy/utils/Initializable.sol

```diff
@@ -153,26 +151,74 @@ abstract contract Initializable {
      * Emits an {Initialized} event the first time it is successfully executed.
      */
     function _disableInitializers() internal virtual {
-        if (_initializing) {
+        // solhint-disable-next-line var-name-mixedcase
+        InitializableStorage storage $ = _getInitializableStorage();
+
+        if ($._initializing) {
             revert AlreadyInitialized();
```

Comment:
That is not changed by this PR, but I'm wondering if that is the right error to trigger.

Reply:
I agree with you but we may decide on this once we decide to tackle #3950 as well